meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-domd.dts

```diff
@@ -45,6 +45,14 @@
 	 * CPU nodes to be present get broken. This is why we have to explicitly
 	 * remove the following.
 	 */
+	/*
+	 * Remove interrupts-extended node from guest domain
+	 * as it can not be shared by Xen.
+	 */
+	pmu_a76 {
+		compatible = "arm,cortex-a55-pmu";
```

Comment:
Do you really need to override `compatible`?

Reply:
if so, I think it is better to completely remove existing node (invalid naming) and provide new with correct name and proper `compatible`.

Reply:
No, we do not need to override `compatible`, it's a copy-paste issue. I removed this line.